tests/QueueTest.php

```diff
@@ -377,10 +369,6 @@ public function testGetJobs()
 
     public function testHugeJob()
     {
-        // Default value - maxBatchSize = 1000
-        setConfig('algoliasearch/queue/number_of_job_to_run', 10);
```

Comment:
That's true, but I'd keep it here said explicitly, because the is dependent on this value. And in case you change default values in XML file, you might break the tests or make it false positive.

The same applies to testMaxSingleJobSize() method.